concordia/views.py

```diff
@@ -375,6 +376,16 @@ class HomeView(ListView):
     )
     context_object_name = "campaigns"
 
+    def get_context_data(self, *args, **kwargs):
+        ctx = super().get_context_data(*args, **kwargs)
+
+        ctx["slides"] = CarouselSlide.objects.published().order_by("ordering")
+
+        if ctx["slides"]:
```

Comment:
We could drop this in favor of using `slides|first` or `slides.0` in the template